with-realm/app/components/IntroText.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {View, Text, Pressable, StyleSheet} from 'react-native';
+// @ts-ignore openURLInBrowser will open the url in your machine browser. (This isn't currently typed in React Native)
+import openURLInBrowser from 'react-native/Libraries/Core/Devtools/openURLInBrowser';
+
+import colors from '../styles/colors';
+
+function IntroText() {
+  return (
+    <View style={styles.content}>
+      <Text style={styles.paragraph}>
+        Welcome to the Realm React Native TypeScript Template
+      </Text>
+      <Text style={styles.paragraph}>
+        Start adding a task using the form at the top of the screen to see how
+        they are created in Realm. You can also toggle the task status or remove
+        it from the list.
+      </Text>
+      <Text style={styles.paragraph}>
+        Learn more about the React Native Realm SDK at:
+      </Text>
+      <Pressable
+        onPress={() =>
+          openURLInBrowser('https://docs.mongodb.com/realm/sdk/react-native/')
```

Comment:
Maybe just Linking.openUrl since the function is untyped

Reply:
Ah, I just remembered why we are using this function.  We basically assume for first time usage the user is using a simulator.  This opens the link on their computers default browser instead of in the simulator browser.  It is, however, a bit of a hack.